flag.go

```diff
@@ -136,7 +136,10 @@ type StringSlice []string
 
 // Set appends the string value to the list of values
 func (f *StringSlice) Set(value string) error {
-	*f = append(*f, value)
+	for _, s := range strings.Split(value, ",") {
```

Comment:
What do you think about making this delimiter optional and specifiable on the `StringSlice` itself? This would allow us to maintain backwards compatibility.